Dockerfile.crossbuild

```diff
@@ -11,9 +12,9 @@ ENV KAFKA_VERSION=$kafka_version \
     PATH=${PATH}:${KAFKA_HOME}/bin
 
 COPY download-kafka.sh start-kafka.sh broker-list.sh create-topics.sh /tmp/
-
-RUN apk add --update unzip wget curl docker jq coreutils \
- && chmod a+x /tmp/*.sh \
+RUN mkdir -p /opt
```

Comment:
I think this and the following `RUN`s could benefit from being a single `RUN`, i.e. one layer less.

Reply:
Will fix in the next patch. Thanks for the comment.